index.html

```diff
@@ -53,6 +53,149 @@ <h1  class="headline1">I'm Loukmane
+              <p>javaScript</p>
+            </div>
+            <div class="view-project">
+            <p>See Project</p>
+          </div>
+            </li>
+            <li class="work work3">
+              <img class="img1" src="portfolioimgs/Snap3.png" alt="work1Snapshot">
+                <h1>Tonic</h1>
+                <div class="canopy">
+                <h3>CANOPY</h3>
+                <ul class="cards">
+                  <li class="info">Back End Dev</li>
+                  <li class="info">2015</li></ul>
+                </div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <div class="tools"> <p>html</p>
+                <p>css</p>
+                <p>javaScript</p>
+              </div>
+              <div class="view-project">
+              <p>See Project</p>
+            </div>
+              </li>
+              <li class="work work4">
+                <img class="img1" src="portfolioimgs/Snap4.png" alt="work1Snapshot">
+                  <h1 class="multi">Multi-Post
+                    Stories</h1>
+                  <div class="canopy">
+                  <h3>CANOPY</h3>
+                  <ul class="cards">
+                    <li class="info">Back End Dev</li>
+                    <li class="info">2015</li></ul>
+                  </div>
+                  <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <div class="tools"> <p>html</p>
+                  <p>css</p>
+                  <p>javaScript</p>
+                </div>
+                <div class="view-project">
+                <p>See Project</p>
+              </div>
+                </li>
+         </ul>
+       </section>
+
+       <section class="about-section">
+        <div class="about">
+         <h1 class="aboutMySelf" >About<br>Myself</h1>
+          <p class="text text2 ">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+<p class="contact">LET'S CONNECT</p>
+<ul class="box">
+<li class="socialMedia">
+    <a href="https://twitter.com/LoukmaneOustani">
+      <img src="portfolioimgs/1.png" alt="twitter">
+     </a>
+ </li>
+ <li class="socialMedia">
+   <a href="https://www.linkedin.com/in/loukmane-oustani-221668211/">
+     <img src="portfolioimgs/2.png" alt="Linkedin">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a href="#">
+     <img src="portfolioimgs/3.png" alt="M">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a href="https://github.com/L2oukmane">
+     <img src="portfolioimgs/4.png" alt="github">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a href="#">
+     <img src="portfolioimgs/5.png" alt="peace">
+   </a>
+ </li>
+</ul>
+        </div>
+<div class="resume">
+  <p>Get my resume</p>
```

Comment:
- [ ] Please ensure the button is a link `<a>` or `<button>` tag instead of using a `p` tag doing this will make your code more semantic.